tests/unit/test_charm_2.py

```diff
@@ -1,9 +1,11 @@
 import os
 import unittest
 import unittest.mock
-
+import ops.pebble
 import ops.testing
+
 from charm import WordpressCharm
+from exceptions import *
```

Comment:
Similar to previous comment, importing `*` is usually considered bad practice: https://www.geeksforgeeks.org/why-import-star-in-python-is-a-bad-idea/

Reply:
See comment above, thanks!